packages/components/tests/dummy/app/components/shw/placeholder/index.js

```diff
@@ -56,6 +56,9 @@ export default class PlaceholderIndexComponent extends Component {
     if (this.args.background) {
       styles.push(`background: ${this.args.background}`);
     }
+    if (this.args.color) {
```

Comment:
@MelSumner if you don't mind, I would like to take this task of making so that the placeholder **by default** uses foreground and background colors that meet contrast criteria. Do you mind waiting to introduce changes to the `Shw::Placeholder` in this PR and wait for my PR to be merged?

Reply:
For context: I was thinking of doing it tomorrow

Reply:
FWIW I did look in the placeholder styles themselves and they are set up to have a conformant contrast but if someone declares a different background color, that's when the strangeness occurs. That's why I thought it might be more flexible to just make it easier to declare a new foreground color if you are declaring a different background color. Hope the context helps! I don't mind whatever we decide to do either way.

Reply:
@didoo so I should remove `@color` support?

Reply:
ok. I'm going to push the changes so you can see the things that fail. I think I might just be missing something that you expect.

Reply:
marking this as resolved